<!--xml-->
<launch>
    <node name="time_slave" pkg="icarus_rover_v2" type="time_slave"  output="screen">
        <param name="loop_rate"             value="10000"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="true"/>    
    </node>
    <node name="time_master" pkg="icarus_rover_v2" type="time_master"  output="screen">
        <param name="loop_rate"             value="10000"/>
	    <param name="verbosity_level"       value="DEBUG"/>
        <param name="require_pps_to_start"  value="false"/>    
    </node>
</launch>
